src/js/utils/time-ranges.js

```diff
@@ -1,16 +1,10 @@
-import log from './log.js';
-
 function rangeCheck(fnName, index, maxIndex) {
   if (index < 0 || index > maxIndex) {
     throw new Error(`Failed to execute '${fnName}' on 'TimeRanges': The index provided (${index}) is greater than or equal to the maximum bound (${maxIndex}).`);
   }
 }
 
 function getRange(fnName, valueIndex, ranges, rangeIndex) {
-  if (rangeIndex === undefined) {
```

Comment:
Yeah, at first I thought the `rangeCheck` would throw, but it won't. I'll add that.